packages/terra-markdown/tests/jest/markdown.test.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Markdown from '../../lib/Markdown';
```

Comment:
Should Jest tests import from src or from lib?

Reply:
yup. https://github.com/cerner/terra-core/pull/1426/commits/6ec9d90d30e92d13cc6af9b5dcada274632063c7